service/src/main/java/com/fecklessweasel/service/UserSessionServlet.java

```diff
@@ -0,0 +1,84 @@
+package com.fecklessweasel.service;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import com.fecklessweasel.service.datatier.SQLSource;
+import com.fecklessweasel.service.datatier.SQLInteractionInterface;
+import com.fecklessweasel.service.objectmodel.ServiceException;
+import com.fecklessweasel.service.objectmodel.ServiceStatus;
+import com.fecklessweasel.service.objectmodel.User;
+import com.fecklessweasel.service.objectmodel.UserSession;
+
+/**
+ * Servlet for creation and deletion of UserSessions (log in and logout)
+ * @author Christian Gunderman
+ */
+@WebServlet("/servlet/user_session")
```

Comment:
Again, would recommend changing to just /user_login or something. Technically every page on the webpage will be a servlet, so identifying it's a servlet feels redundant.